test/k8sT/Updates.go

```diff
@@ -11,6 +11,20 @@ import (
 )
 
 var _ = Describe("K8sUpdates", func() {
+
```

Comment:
You can do whatever you want, but my advice here wil be the following: 

- Have two separate test, if not.. the test that always fail is the Upgrade one, and It'll be a mess. 
- That description in the top of the describe can be done with By messages, and people will understand the test as it runs. 

But as I said, that can be kept, and feel free to merge this one. 
Regards.

Reply:
The I've added more By's. I'll keep this description as it is easier to have an overview of what the test is suppose to do instead of reading all 250 lines.

WRT the 2 tests, it's really difficult to split both tests as one depends on the other. The cilium state present after the upgrade should and needs to affect the downgrade test.

Reply:
I mean, as I said, I'm happy to keep as you think that it's better, but I also read few weeks back(In slack)you said that the downgrade test can fail, but the update will never fail, so we don't really now if the update works correct and we can have a lot of false positives. 

TBH: I would keep the Update test as it is, and add a extensive and longer test (With more policies, CIDR policies, background connectivity, different pods with init containers) in the nightly test, that will help us to be close to the users world, with more nodes ( Nightly has 4 nodes) , different services, etc.. 

Regards